spec/config/initializers/request_logging_spec.rb

```diff
@@ -5,8 +5,8 @@
         # rubocop:disable RSpec/AnyInstance
         allow_any_instance_of(ActionController::TestRequest).
           to receive(:headers).
-          and_wrap_original do |request|
-            headers = request.call
+          and_wrap_original do |original_headers_method|
+            headers = original_headers_method.call
```

Comment:
(Unrelated change; I just noticed that the block parameter name here is confusing; I am guessing it's some sort of copy-paste carelessness.)